pkg/settings/setting.go

```diff
@@ -95,7 +95,7 @@ var (
 	IgnoreNodeName                    = NewSetting("ignore-node-name", "") // nodes to ignore when syncing v1.node to v3.node
 	NoDefaultAdmin                    = NewSetting("no-default-admin", "")
 	RestrictedDefaultAdmin            = NewSetting("restricted-default-admin", "false") // When bootstrapping the admin for the first time, give them the global role restricted-admin
-	EKSUpstreamRefreshCron            = NewSetting("eks-refresh-cron", "*/5 * * * *")
+	ClusterUpstreamRefreshCron        = NewSetting("eks-refresh-cron", "*/5 * * * *")
```

Comment:
I know you are addressing this later, but to be clear we'll need to revert this as either of our solutions leave eks with its own refresh setting.